pkg/collector/corechecks/snmp/internal/report/report_bandwidth_usage.go

```diff
@@ -97,18 +100,69 @@ func (ms *MetricSender) sendBandwidthUsageMetric(symbol profiledefinition.Symbol
 	}
 	usageValue := ((octetsFloatValue * 8) / (float64(ifSpeed))) * 100.0
 
+	interfaceID := ms.deviceID + ":" + fullIndex + "." + usageName
+	rate, err := ms.interfaceBandwidthState.calculateBandwidthUsageRate(interfaceID, ifSpeed, usageValue)
```

Comment:
(minor) instead of building the interfaceID in `sendBandwidthUsageMetric`, I think it might be better to move this interfaceID construct inside `calculateBandwidthUsageRate` by passing all the required info.

The rational is to avoid leaking this logic outside of `InterfaceBandwidthState` part of the code when possible.